index.js

```diff
@@ -440,10 +469,12 @@ module.exports = postcss.plugin('postcss-modules-local-by-default', function(
           atrule.params = localMatch[0];
           globalKeyframes = false;
         } else if (!globalMode) {
-          atrule.params = ':local(' + atrule.params + ')';
+          if (!localAliasMap.has(atrule.params))
```

Comment:
Some plugins for postcss remove `atrule.params` can we add check on `atrule.params`